_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -60,23 +60,23 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 
 #### Passed Example 1
 
-`aria-required` property with valid true/false value
+This `aria-required` attribute has a valid true/false value.
 
 ```html
 <div role="textbox" aria-required="true" aria-label="A required textbox"></div>
 ```
 
 #### Passed Example 2
 
-`aria-expanded` state with valid true/false/undefined value
+This `aria-expanded` attribute has a valid true/false/undefined value.
 
 ```html
 <div role="button" aria-expanded="undefined">A button</div>
 ```
 
 #### Passed Example 3
 
-`aria-pressed` state with valid tristate value
+This `aria-pressed` attribute has a valid true/false/mixed/undefined value.
```

Comment:
```suggestion
This `aria-pressed` attribute has a valid tristate value.
```
"tristate" is the correct ARIA term for this attribute: https://www.w3.org/TR/wai-aria-1.2/#aria-pressed